importApp/src/main/java/com/commercetools/dataimport/orders/OrderCsvLineValue.java

```diff
@@ -0,0 +1,101 @@
+package com.commercetools.dataimport.orders;
+
+
+/**
+ * This class is used as a container for the csv line data
+ */
+
+public class OrderCsvLineValue {
+
+    private String customerEmail;
+    private String orderNumber;
+    private LineItems lineItems;
+    private double totalPrice;
+
```

Comment:
Actually this class is used just for mapping CSV line to a java object so this is why i mad the classes inner just just to avoid adding many java files since they serve one small purpose.
For the code  formatting i agree 👍